modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss

```diff
@@ -63,6 +63,14 @@ $tab-header-height: 38px;
 	margin: 0;
 	padding: 0.2rem;
 
+	@include media-breakpoint-down(xs) {
+		@at-root {
+			.mobile #{&} {
+				padding-bottom: 7rem;
```

Comment:
![bleblu](https://user-images.githubusercontent.com/905006/87560846-2bd8c800-c6bc-11ea-8a2b-8466f69b397d.gif)

Reply:
Just in case the gif is not clear... what does `.mobile #{&}` even mean? 😂 

Just out of curiosity :)

Reply:
The `#{&}` stands for the first class in the section
```scss
.lorem {
   .ipsum {
        .mobile #{&} {...
```
Will be
```css
.lorem .ipsum .mobile .lorem {...
```
We use it to avoid repeating the name of the parent class but since we don't want the `.mobile` class to be a child of the others we add `@at-root` to exit the inheritance and obtain 
```css
.mobile .lorem {...
```

Reply:
We don't need to interpolate (`#{}`) the Sass parent selector (`&`) here. We only need to interpolate when we want the selector `.mobile.global-apps-nav` (`.mobile#{&}`).

Personally, I don't like using `.selector &` or `@at-root` unless I have to (e.g., in a mixin). It's hard to follow.

```
.mobile .global-apps-nav {
  @include media-breakpoint-down(xs) {
    padding-bottom: 7rem;
  }
}
```
is easier to read IMO.

Reply:
@eduardoallegrini brings up an interesting Sass feature?

|selector|output|
|---|---|
|`.mobile &`|`.mobile .global-apps-nav`|
|`.mobile #{&}`|`.global-apps-nav .mobile .global-apps-nav`|
|`@at-root { .mobile #{&} }`|`.mobile .global-apps-nav`|
|`.mobile#{&}`|`.global-apps-nav .mobile.global-apps-nav`|
|`@at-root { .mobile#{&} }`|`.mobile.global-apps-nav`|

Reply:
I think Marcos wanted to keep `.global-apps-nav` as the outer class and `.mobile` as "external modifier" 😆 (I don't mind)

Reply:
I thought about `.mobile &` too, but does it work with the mixin `@include media-breakpoint-down(xs) {` in the way?

Reply:
It works, the only thing IIRC that doesn't work in media queries is `@extend`.

Reply:
Thanks for the cookbook @pat270!

I still understand we're good to forward this? or do you guys want to make any changes?

Reply:
Wanna hit the forward button, then? 😉